source/includes/_enrichment.md

```diff
@@ -435,6 +435,7 @@ hashed_email (optional) | The person's md5 or sha256 hashed email | 8d935115b9ff
 organization_name (optional) | The person's company name | Apollo Inc.
 domain (optional) | The person's company domain | apollo.io
 id (optional) |  The person's ID obtained from the search endpoint | "583f2f7ed9ced98ab5bfXXXX"
+linkedin_url (optional) | The person's linkedin URL | http://www.linkedin.com/in/tim-zheng
```

Comment:
Can you add this param in the request body of the sample example given in shell and python for the customers to know how they need to pass linkedin_url?

Reply:
If we do that, the response that we receive will change from the sample curl request. Do we want that?
Also, `linkedin_url` is sufficient to work as an input of its own. Added a point [here](https://www.notion.so/apolloio/RCE-API-Documentation-Auditing-people-match-and-people-bulk_match-619f131cdd1949e9bdaae9ef7af2108e?pvs=4#09ffc446ee194a528e2809cc20a37595) to address this problem of groups. Waiting on deciding on solution for this from product.

Reply:
Sure, will do that.

Reply:
@atul-tw , added in request and also updated the sample response with the updated new responses.